BuildResidentialHPXML/measure.rb

```diff
@@ -3452,13 +3522,22 @@ def self.set_walls(hpxml, runner, model, args)
 
       next if exterior_adjacent_to == HPXML::LocationLivingSpace # already captured these surfaces
 
+      attic_wall_type = nil
+      if ([HPXML::LocationAtticUnvented, HPXML::LocationAtticVented].include? interior_adjacent_to) && (exterior_adjacent_to == HPXML::LocationOutside)
+        attic_wall_type = HPXML::AtticWallTypeGable
+      end
+
       wall_type = args[:wall_type]
       if [HPXML::LocationAtticUnvented, HPXML::LocationAtticVented].include? interior_adjacent_to
         wall_type = HPXML::WallTypeWoodStud
       end
 
       if exterior_adjacent_to == HPXML::LocationOutside && args[:wall_siding_type].is_initialized
-        siding = args[:wall_siding_type].get
+        if ([HPXML::LocationAtticUnvented, HPXML::LocationAtticVented].include? interior_adjacent_to) && (args[:wall_siding_type].get == HPXML::SidingTypeNone)
+          siding = HPXML::SidingTypeVinyl
```

Comment:
This is not an appropriate change. If the user doesn't choose a siding, we need the resulting HPXML to reflect that that input wasn't provided. (That doesn't mean the building will have no siding, just that it will be defaulted downstream in the HPXMLtoOpenStudio measure.) 

I'm not sure what you're trying to accomplish here. Can you explain?

Reply:
This is meant to prevent wood stud gable walls from having `None` siding. This occurs, for example, when brick/CMU wall siding is set to `None`, but gable walls are forced to be wood stud in this measure. It is supposed to mimic what is done in `resstock`: https://github.com/NREL/resstock/blob/develop/resources/measures/ResidentialConstructionsWallsCMU/measure.rb#L191-L196  

So instead of assigning the siding here, should it get assigned here instead? https://github.com/NREL/OpenStudio-HPXML/blob/build-res-hpxml-v3-hes/HPXMLtoOpenStudio/measure.rb#L667

Reply:
I guess my recommendation would be to use `siding = nil` here. That will create an HPXML file where the gable walls have no siding specified -- which will then get defaulted downstream in the HPXMLtoOpenStudio measure to wood siding. It's not vinyl siding, but it's close enough.

Reply:
Okay that makes sense, however this will end up causing an error with the hescore-hpxml translator, which is what prompted the change originally. Should I set this value in hpxml_defaults so that it is valid in `in.xml`?

Reply:
I thought hescore-hpxml completely ignores gable walls?

Reply:
If I try to run a building with a wood stud wall and no siding (gable wall or otherwise) it will throw a translation error

Reply:
Okay, but the code above is only addressing gable walls.

Reply:
Yes, I get a translation error when running with a wood stud gable wall and no siding

Reply:
It sounds like hescore-hpxml needs to be fixed to ignore gable walls then.